### ISSUE TITLE
Update website

### ISSUE THREAD

**Issue by Kebap** [Mudlet/VideoPlanning#10]
Need to review https://www.mudlet.org/media/ eventually

**Comment by Kebap** [comment]
This was referring to the listed videos on that page. They seem quite dated (Mudlet 1 or 2 at best?)
Review entails, checking the existing videos' content and matching it with the planned video series.
The video series should be promoted there for sure, only maybe some old videos keep merit as well.

**Comment by vadi2** [comment]
Agreed! I don't mind my old videos going away to make place for newer ones.
